unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogService/WebInterfaceEmotesCatalogService.cs

```diff
@@ -172,6 +174,10 @@ public async UniTask<IReadOnlyList<WearableItem>> RequestOwnedEmotesAsync(string
             return promise.value;
         }
 
+        public UniTask<IReadOnlyList<WearableItem>> RequestEmoteCollectionAsync(IEnumerable<string> collectionIds,
+            CancellationToken cancellationToken, List<WearableItem> emoteBuffer = null) =>
+            throw new NotImplementedException("Implemented in LambdasEmotesCatalogService");
```

Comment:
same concern here

Reply:
`WebInterfaceEmotesCatalogService` and `WebInterfaceWearablesCatalogService` are marked as obsolete and could be removed after this implementation. They are only being used to preview wearables & emotes in the backpack v1. This implementation is deprecating that flow. So i think its over-engineering at this point to segregate into new interfaces.